api/core/v3/pipeline.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+import "github.com/gogo/protobuf@v1.3.1/gogoproto/gogo.proto";
+import "github.com/sensu/sensu-go/api/core/v2/meta.proto";
+
+package sensu.core.v3;
+
+option go_package = "v3";
+option (gogoproto.populate_all) = true;
+option (gogoproto.equal_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.testgen_all) = true;
+
+// Pipeline represents a named collection of pipeline workflows.
+message Pipeline {
+  // Metadata contains the name, namespace, labels and annotations of the
+  // pipeline.
+  sensu.core.v2.ObjectMeta Metadata = 1 [ (gogoproto.jsontag) = "metadata", (gogoproto.nullable) = true ];
+
+  // Pipelines contains one or more pipelines.
```

Comment:
I think this should read "Workflows contains one or more pipelines"?

Reply:
"Pipelines contain one or more workflows".

Reply:
Updated to `Workflows contains one or more pipeline workflows`.